gritulator/control/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""This package contains example controllers."""
+
+from gritulator.control._common import (
+    ComplexFFPICtrl,
+    ComplexPICtrl,
+    RateLimiter,
+    DCBusCtrl,
+    PICtrl,
+    PWM
+    )
+
+import gritulator.control.grid_following as gfl
```

Comment:
They are used in the example files.

Reply:
If only there, would it be more clear if the examples uses grid_following instead of gfl and grid_forming instead of gfm? Those longer names are easier to understand.

Reply:
Fixed.